Hardware/IA32.md

```diff
@@ -29,7 +29,7 @@ menuentry "Load seL4 VM" --class os {
     insmod part_msdos
     insmod ext2
     set root='(hd0,msdos2)'
-    multiboot /boot/sel4kernel
+    multiboot /boot/seL4
```

Comment:
this one looks like it got caught in a global search/replace. It should stay on `sel4kernel`